media_kit/pubspec.yaml

```diff
@@ -55,3 +56,6 @@ ffigen:
     entry-points:
       - headers/client.h
   dart-bool: true
+flutter:
```

Comment:
package:media_kit is a Dart (not Flutter) package. Does this work as intended?

NIT: Add a new line above it.